src/main/java/io/hops/metadata/ndb/NdbStorageFactory.java

```diff
@@ -52,6 +53,7 @@
 import io.hops.metadata.ndb.dalimpl.yarn.rmstatestore.DelegationKeyClusterJ;
 import io.hops.metadata.ndb.dalimpl.yarn.rmstatestore.DelegationTokenClusterJ;
 import io.hops.metadata.ndb.mysqlserver.MysqlServerConnector;
+import io.hops.metadata.yarn.dal.AppProvenanceDataAccess;
```

Comment:
It seems you missed to import the other two classes
FileProvenanceDataAccess and FileProvXAttrBufferDataAccess and the their clusterj classes

Reply:
They are imported by 
import io.hops.metadata.hdfs.dal.*;

Reply:
there is an import for all the hdfs DALs

https://github.com/hopshadoop/hops-metadata-dal-impl-ndb/blob/acada8c148133fff5c8df49a5c341aadd8d4e86e/src/main/java/io/hops/metadata/ndb/NdbStorageFactory.java#L28